src/wdio/browser.ts

```diff
@@ -127,4 +127,17 @@ export class Browser {
         const caps = browser.capabilities as any;
         return caps.platformName;
     }
+
+    // Window
+    public static async getWindowSize(): Promise<void> {
+        await browser.getWindowSize();
```

Comment:
If this method is a getter, it should return something (probably an object with 2 integers, width and height). Otherwise, it will be completely useless.

Reply:
Indeed. added